pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -1634,7 +1634,7 @@ class ClassTypeHintMatch:
                 ),
                 TypeHintMatch(
                     function_name="repeat",
-                    return_type=["str", None],
+                    return_type=["RepeatMode", None],
```

Comment:
```suggestion
                    return_type=["RepeatMode", "str", None],
```
Should this include `str`?

Reply:
This section is not blocking (shows in IDE but bypassed in CI). I think it's better to not have `str`
Which makes me think that perharps we can also drop `str` in `state` on line 1508

Reply:
Didn't realize that the plugin is completely ignore in CI runs and pre-commit.

Ok, let's do it then. Should the fallback `str` case be removed from other functions, too? E.g. `media_content_type` and `device_class`? Maybe that could also be done separately, later.

Reply:
The doc states that

> Using the integration name as the `media_content_type` is also acceptable within the play_media service if the integration provides handling which does not map to the defined constants.

And for `device_class` we usually also allow custom device classes.

Reply:
👍🏻